packages/react-native-editor/__device-tests__/gutenberg-editor-gallery.test.js

```diff
@@ -34,6 +34,11 @@ describe( 'Gutenberg Editor Gallery Block tests', () => {
 
 	it( 'should be able to add a gallery block', async () => {
 		await editorPage.addNewBlock( galleryBlockName );
+		await driver.sleep( 1000 );
+		await editorPage.closePicker();
+		// wait for the Picker to be closed
+		await driver.sleep( 1000 );
```

Comment:
@guarani I am going to remove this tomorrow and see if it's really required because I am not totally sure because I was getting some CI failures earlier on. Now I am better able to interpret the error messages so I will give it a try. But if you are able to you could also remove/ re-add them in your local environment and see if the test pass.

Reply:
So the first `await driver.sleep( 1000 );` was necessary to wait until the bottom sheet is shown after a new block is added but the second one wasn't. The image tests were also failing so I added `await driver.sleep( 1000 );` to that test as well and closed the picker there too and all should be fine now f217d4a